### PR TITLE
Add highlight

### PR DESCRIPTION
**Pull request by morizotter**
Add highlighted background color.

### CODE REVIEW

File: TagListView/TagView.swift [hunk 1]
```diff
@@ -51,6 +51,12 @@ public class TagView: UIButton {
         }
     }
     
+    @IBInspectable public var tagHighlightedBackgroundColor: UIColor = UIColor.blueColor() {
+        didSet {
+            backgroundColor = highlighted ? tagHighlightedBackgroundColor : tagSelectedBackgroundColor
```
Comment (xhacker) [inline TagListView/TagView.swift]: Is this a mistake? I think `tagSelectedBackgroundColor` should be `tagBackgroundColor` here.

**Comment by xhacker** [comment]
Thanks for the pr! Looks mostly good to me, found a small issue.

**Comment by morizotter** [comment]
Sorry, I missed your reply. Anyway, thanks for merge it ;)

**Comment by xhacker** [comment]
No worries. Thanks for your contribution!

**Comment by greggmojica** [comment]
how can i get the highlight color to stay using this library?

**Pull request merged**